src/models/DefaultFilters.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+const ElectronConfig = require('electron-config')
+const configName = process.env.NODE_ENV === 'test' ? 'config-test' : 'config'
+const storage = new ElectronConfig({ name: configName })
+const Filter = require('./Filter')
+const LastFilter = require('./LastFilter')
+const DEFAULT_FILTER_KEY = 'default-filters'
+
+class DefaultFilters {
+  static init(login) {
+    if (!storage.has(DEFAULT_FILTER_KEY)) {
+      this.myIssuesFilter(login)
+      this.myCommentsFilter(login)
+      this.myAssignmentsFilter(login)
+      this.myMentionsFilter(login)
+      this.myPopularFilter(login)
+      this.save()
+    }
+    return
+  }
+
+  static myIssuesFilter(login) {
+    const defaultFilter = new Filter('My Issues')
+    defaultFilter.store(`author:${login} is:open sort:updated-desc`)
+    LastFilter.save(defaultFilter.key)
+  }
+
+  static myCommentsFilter(login) {
+    const defaultFilter = new Filter('My Comments')
+    defaultFilter.store(`commenter:${login} sort:updated-desc`)
```

Comment:
I was trying to think of ways of DRY-ing it up as I saw all those similar methods, but wondered how to keep the default filter list editable enough. I like this, it makes it clear what each filter is, and gives a good place to add more without making this file too long.